mathics/builtin/structure.py

```diff
@@ -1276,5 +1281,10 @@ class ByteCount(Builtin):
 
     def apply(self, expression, evaluation):
         'ByteCount[expression_]'
-
-        return Integer(count_bytes(expression))
+        if bytecount_support == False:
```

Comment:
I thought it was just PyPy that didn't support bytecount?
In that case:
```
if platform.python_implementation == 'PyPy':
    bytecount_supporty = False
else:
    import ...
    bytecount_support = True
```
and then
```
if not bytecount_support:
    return evaluation.message('ByteCount', 'pypy')
```
would be simpler.